deploy/roles/eod/tasks/main.yaml

```diff
@@ -245,3 +252,163 @@
   when: custom_repo
   tags:
     - eod
+
+- name: Create Xconfig directory 
+  file: 
+    path: "{{ eod_Xconfig_directory }}"
+    state: directory
+    mode: 0755
+  tags: 
+    - eod 
+
+- name: Set /etc/gdm/gdm.schema file
+  template:
+    src: gdm.schemas.j2
+    dest: /etc/gdm/gdm.schemas
+    mode: 0644
+  notify:
+    - restart eod 
+  tags:
+  - eod
+
+- name: Install yum packages
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - pixmap-miscfixed-font
+    - pixmap-fixed-fonts
+    - abyssinica
+    - tree 
+  when: custom_repo                             ####  ??????
```

Comment:
@olivermel This when statement isn't needed here, as these packages are not in our custom repository.  They should be available in Vagrant, for instance.